server/README.md

```diff
@@ -62,7 +65,13 @@ psql -c 'create database dfac'
 
 (`createdb` if this fails)
 
-2. Create the game_events table
+2. Create the tables
+
+```
+./create_fresh_dbs.sh
```

Comment:
thanks for setting this up! This will definitely be really helpful as new contributors onboard to the server code.